testing/adapter/index.js

```diff
@@ -24,11 +39,12 @@ function getEthErc20Balance ({ ethAccountAddress, ethNodeUrl, ethErc20Address, e
     JSON.parse(fs.readFileSync(ethErc20AbiPath)),
     remove0x(ethErc20Address)
   )
-  return ethContract.methods.balanceOf(remove0x(ethAccountAddress)).call().then(result => {
+  ethContract.methods.balanceOf(remove0x(ethAccountAddress)).call().then(result => {
```

Comment:
Why did you remove `return`? This will effectively mean that the async calls will get executed in background and you won't be able to await on the result from this function. Why do we need to explicitly call `exit()` here?

Reply:
Described in separated comment above.
`return` is not needed because the result of the function is void.